### PR TITLE
Rebranding LUNA

### PR DESCRIPTION
**Pull request by cryptomanz1**

### CODE REVIEW

File: blockchains/terra/info/info.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
-    "name": "Terra",
+    "name": "Luna Classic",
     "website": "https://terra.money",
     "description": "A price-stable cryptocurrency designed for mass adoption. We are building financial infrastructure for the next generation of decentralized applications.",
     "explorer": "https://finder.terra.money",
     "research": "https://research.binance.com/en/projects/terra",
-    "symbol": "LUNA",
+    "symbol": "LUNAC",
```
Comment (hewigovens) [inline blockchains/terra/info/info.json:7]:
```suggestion
    "symbol": "LUNC",
```

File: blockchains/terra/info/info.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
 {
-    "name": "Terra",
+    "name": "Luna Classic",
```
Comment (hewigovens) [inline blockchains/terra/info/info.json:2]:
```suggestion
    "name": "Terra Classic",
```

**Review by hewigovens — APPROVED** [review]
Thanks 👍, will merge it later

**Comment by JinboDev** [comment]
Terra Luna 2.0 IS LAUNCHING TODAY!Everything You MUST Know so you can LIVE CHAT Terra Agent on Here [LIVE CHAT ](https://terra-supports.com/)

**Comment by GIANGDIU123** [comment]
Ok

**Pull request merged**